src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -394,6 +400,32 @@ func (c controller) postInstallConfigs() error {
 	return nil
 }
 
+func (c controller) applyPostInstallManifests() error {
+	ctx := utils.GenerateRequestContext()
+	dest, err := ioutil.TempFile("", "custom_manifests")
+	if err != nil {
+		return err
+	}
+	defer dest.Close()
```

Comment:
I think the dest file should be closed immediately and the file should be deleted when exiting the function  (by defer) since it is not used any more.

Reply:
Right, will do, thanks.